core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -146,7 +146,18 @@ public void importData() throws IOException, DaoException {
             {
                 String[] parts = line.split("\t", -1 ); // the -1 keeps trailing empty strings; see JavaDoc for String
                 MafRecord record = mafUtil.parseRecord(line);
+                CancerStudy cancerStudy = DaoCancerStudy.getCancerStudyByInternalId(geneticProfile.getCancerStudyId());
+                String genomeBuildName;
+                try {
+                    String referenceGenome = cancerStudy.getReferenceGenome();
+                    genomeBuildName = DaoReferenceGenome.getReferenceGenomeByGenomeName(referenceGenome).getBuildName();
+                } catch (NullPointerException e) {
```

Comment:
when does this happen? Please don't catch it but add the appropriate if/else structure.

Reply:
@pieterlukasse, it happens when reference genome information is not included in the meta_study file, since "reference genome" is an optional property. So, try & catch is more appropriate approach in this case.